v2/thirdparty/pre-built-ui/multitenant-login.mdx

```diff
@@ -0,0 +1,130 @@
+---
+id: multitenant-login
+title: "Multitenant login"
+hide_title: true
+---
+
+import AppInfoForm from "/src/components/appInfoForm"
+import BackendSDKTabs from "/src/components/tabs/BackendSDKTabs";
+import GoFrameworkSubTabs from "/src/components/tabs/GoFrameworkSubTabs";
+import TabItem from '@theme/TabItem';
+import MultitenancyIntro from '../reusableMD/multitenancy/intro.mdx'
+import CreateConfigureTenants from '../reusableMD/multitenancy/create-configure-tenants.mdx'
+import BackendCORS from '../reusableMD/multitenancy/backend-cors.mdx'
+
+:::important
+TODO BANNER This is a paid feature
+
+If you wish to use this feature for free, follow this implementation guide TODO link
+:::
+
+# Multitenant login
+
+<MultitenancyIntro />
+<CreateConfigureTenants />
+
+## Step 2: Frontent setup
+
+Modify the `SuperTokens.init` to do the following:
+1. Set the `usesDynamicLoginMethods` to true and remove the providers list
+2. Initialize the `Multitenancy` recipe and provide `getTenantId` config function
+
+<AppInfoForm
+    askForAppName
+    askForAPIDomain
+    askForWebsiteDomain
+>
+
+```tsx
```

Comment:
use the right components - angular and vue should also be there.. see other pages in this section